### PR TITLE
Reload function base on CONFIG_PATH

### PR DESCRIPTION
**Pull request by maxxnino**

### CODE REVIEW

**Comment by Tastyep** [comment]
Could you clean up your commit history, the pull request contains 7 commits but only the last one is required

**Comment by maxxnino** [comment]
Thank you for good tip, i'm noob with git

**Comment by maxxnino** [comment]
It look the same 😕. Is it works.

**Comment by Tastyep** [comment]
> It look the same confused. Is it works.

No, you even added two more commits (check the commits tab):
Rebasing documentation:
https://git-scm.com/book/en/v2/Git-Branching-Rebasing

**Comment by maxxnino** [comment]
thank you, i will try apply new knowledge, in the future pull request

**Comment by Tastyep** [comment]
What about this one ?

**Comment by maxxnino** [comment]
Nice, i need to drop un-use commit and only pick the last one. And i need to use:
> git push --force origin dev

File: lua/lv-utils/init.lua [hunk 1]
```diff
@@ -1,9 +1,9 @@
 local lv_utils = {}
 
 function lv_utils.reload_lv_config()
-  vim.cmd "source ~/.config/nvim/lv-config.lua"
-  vim.cmd "source ~/.config/nvim/lua/plugins.lua"
-  vim.cmd "source ~/.config/nvim/lua/lv-neoformat/init.lua"
+  vim.cmd("source " .. CONFIG_PATH .."/lv-config.lua")
```
Comment (Tastyep) [inline lua/lv-utils/init.lua:4]: One last thing you need to do:
- Install [stylua](https://github.com/johnnymorganz/stylua)
- Possibly copy utils/.stylua.toml in the root director (~/.config/nvim) <- I don't know if necessary
- Save your code to reformat automatically

This time you can `rebase -i`, change pick to e (for edit), `git add -u` and then `git rebase --continue` + `git push -f`.
Alternatively you can also `git reset --soft HEAD^`, add and commit, then `git push -f`

**Review by Tastyep — CHANGES_REQUESTED** [review]

File: lua/lv-utils/init.lua (hunk 1 above)
Comment (maxxnino) [reply lua/lv-utils/init.lua]: I applied stylua, everything look nice and clean.

**Review by jameswalmsley — APPROVED** [review]

**Review by Tastyep — APPROVED** [review]

**Review by jameswalmsley — APPROVED** [review]

**Pull request closed without merging**